Src/Base/AMReX_GpuDevice.cpp

```diff
@@ -343,6 +353,9 @@ Device::Initialize ()
             amrex::Print() << "HIP initialized.\n";
         }
     }
+#if defined(AMREX_USE_ROCTX)
+    roctracer_start();
```

Comment:
For some reason, we call `profileStart` twice in `ifdef AMREX_USE_CUDA`.  I assume that's uncessary. (@maxpkatz)  So I suggest we move `roctracer_start` into `profileStart`, remove the two calls to `profileStart` in `#ifdef AMREX_USE_CUDA`, and put a call to `profileStart` at the end of `Device::Initialize`.

Reply:
I don't know why it's there twice but I agree that we only need one of those two., and it makes sense to put it at the end of Initialize.

Reply:
I might not be remembering this right, but did we try to put a nvtx region around the Initialization to ID if there was unneeded start-up at some point? Is that why it's in there twice, we tried to move the init back to allow wrapping Init?

Reply:
There are only calls to Print() between the two calls to cudaProfilerStart.